src/components/HomeHeader.tsx

```diff
@@ -38,14 +42,15 @@ const HomeHeader: React.FC<{
     }
     const currMode = await schoolUtil.getCurrMode();
     setStudentMode(currMode);
-    DEFAULT_HEADER_ICON_CONFIGS.forEach((element) => {
+    DEFAULT_HEADER_ICON_CONFIGS.forEach(async (element) => {
       // console.log("elements", element);
 
       console.log(currMode);
       if (
         !(
-          currMode === MODES.SCHOOL &&
-          element.headerList === HOMEHEADERLIST.ASSIGNMENT
+          (currMode === MODES.SCHOOL && element.headerList === HOMEHEADERLIST.ASSIGNMENT) ||
+          (await canShowSuggestions === false && element.headerList === HOMEHEADERLIST.SUGGESTIONS)
```

Comment:
you don't need to check like this canShowSuggestions === false just put canShowSuggestions